src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -360,6 +360,43 @@ private static IEnumerable<CompletionItem> GetUserDefinedTypeCompletions(Semanti
             null => null,
         };
 
+        private static string? TryGetSkippedTokenText(SkippedTriviaSyntax skippedTrivia)
+        {
+            // This method attempts to obtain text from a skipped token - in cases where the user has partially-typed syntax
+            // but may be looking for completions.
+            if (skippedTrivia.Elements.Length != 1 ||
```

Comment:
What is the significance of `skippedTrivia.Elements.Length` being `1`?

Reply:
Great question! `SkippedTriviaSyntax` is used by the parser to represent 'unexpected' pieces of syntax, and has the ability to represent multiple - for example, in the following example, we don't expect multiple strings after `'abc'` in a resource declaration:

```bicep
resource foo 'abc' def ghi jkl mno
```

Completions are a bit of a moving target - there's no perfect solution in some cases, it's just about making a best effort attempt to predict what the user is looking for. There are some complexities in handling multiple piece of skipped syntax, with no obvious perfect solution, so I've just opted not to support it here.